library/ssl_tls.c

```diff
@@ -5261,6 +5261,13 @@ int mbedtls_ssl_get_handshake_transcript( mbedtls_ssl_context *ssl,
 #endif /* MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA*/
 
     default:
+#if !defined(MBEDTLS_HAS_ALG_SHA_384_VIA_MD_OR_PSA_BASED_ON_USE_PSA) && \
+    !defined(MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA)
+    (void) ssl;
```

Comment:
I'm confused how to this is necessary: we're apparently building the TLS 1.2 code in a configuration which doesn't have a running hash, and so no connection is possible? That's a preexisting issue (I noticed this earlier when looking at how the `legacy_or_psa.h` macros are used) and the change here is not harmful, so now is not the time to resolve this.